docs/reference/mapping/types.asciidoc

```diff
@@ -126,6 +126,31 @@ the <<analysis-standard-analyzer,`standard` analyzer>>, the
 This is the purpose of _multi-fields_. Most field types support multi-fields
 via the <<multi-fields>> parameter.
 
+[discrete]
+[[types-runtime]]
+=== Runtime
+{es} indexes most field types by default to promote faster search. However,
```

Comment:
In most scenarios the best way to use data in {es} is to index it in a schema, because indexing will provide the best query time performance. However, sometimes users want the flexibility to add fields after the documents were already ingested into {es}. Runtime fields enables users to add fields to documents that are already in {es} without needing to re-index. A runtime field includes instructions for calculating its value, and the value itself is calculated only at the time a query requires it. Runtime fields do not increase the index size and allow for faster ingest process, because a runtime field is not indexed.